labrad/test/test_types.py

```diff
@@ -343,5 +344,9 @@ def testEvalDatetime(self):
         data2 = T.evalLRData(repr(data))
         self.assertEquals(data, data2)
 
+    def testUnicodeStr(self):
```

Comment:
I would suggest calling this something like `testBytesType` or something, since it doesn't have anything to do with unicode yet.